docs/docs/core.ts

```diff
@@ -0,0 +1,143 @@
+import hljs from 'highlight.js/lib/core';
+import python from 'highlight.js/lib/languages/python';
+import components from "streamsync-ui/components.codegen.json";
+
+// @ts-ignore
+hljs.registerLanguage('python', python);
+
+export function highlightCode(code: string, language = 'python') {
+	if (!code) return "";
+
+
+	let value = hljs.highlight(code,{
+		'language': language
+	}).value;
+
+	value = value.replace(/\t/g, "  ")
+	return value
+}
+
+/**
+ * List the different categories of components
+ */
+export function categories() {
+	const categoriesAll = Object.entries(categoriesList).map(([name, description]) => {
+		return name;
+	});
+
+	return categoriesAll;
+}
+
+/**
+ * Get the description of a category
+ * @param name Name of the category
+ */
+export function categoryDescription(name: string) {
```

Comment:
I mean comments like this, whats the purpose of them? 
This is how I see them:
<img width="504" alt="Screenshot 2024-03-27 at 11 53 34" src="https://github.com/streamsync-cloud/streamsync/assets/649972/02118ed3-05d3-4fff-997e-6fb474eb07fb">
😅 
But maybe there is some hidden reason for them?